### DISCUSSION TITLE
Localization files for tinymce editor

### DISCUSSION THREAD

**Discussion by aukc1970** [givanz/Vvveb#233] [General]
Good afternoon! I wanted to ask if it is possible to add localization files for tinymce editor maybe from here https://github.com/mklkj/tinymce-i18n or from the official website.

**Comment by aukc1970** [comment]
Hello Givanz,
Please localize tinymce editor or write instructions on how to do it yourself. Everywhere in the entire CMS there is localization, but in tinymce editor there is no localization, although initially on the official website of tinymce editor there is localization.

**Comment by givanz** [comment]
Hi

Tinymce localization was added in `latest.zip` update, language files are located in `/public/js/libs/tinymce-dist/langs`

**Reply by aukc1970** [reply]
Thank you very much!
